Documentation/src/fp/overview.md

```diff
@@ -0,0 +1,8 @@
+# Functional Programming
+
+* No side effects
+* Pure functions
```

Comment:
Inconsistent capitalization in this list

Reply:
Fixed in 239b3a73af300f846afba75f4a992650b6fcb41b